other/apidsl/tox.in.h

```diff
@@ -1543,6 +1546,47 @@ namespace file {
      * has no avatar.
      */
     AVATAR,
+
+    /**
+     * File type for a "small" image in png format. Sticker are inline images similar in intent to emoji, but should be
+     * their own 'message' and shouldn't share a line with existing or later text. In contrast with standard inline
+     * images, stickers should be made available by the client, static, and multi use. Allowing other clients to save
+     * bandwidth by storing stickers for later use.
+     *
+     * This image should have a file_id generated by `tox_hash` to allow clients to save this image file for reuse, and
+     * later reference. Because stickers are expected to be saved, receiving clients should send TOX_FILE_CONTROL_CANCEL
+     * when they already have the sticker indicated by file_id; and because of this sending clients can't make any
+     * assumptions when these file transfers are canceled with no data transferred.
+     */
+    STICKER_PNG,
+
+    /**
+     * File type for a "small" image in SVG format. Otherwise same as TOX_FILE_KIND_STICKER_PNG.
+     */
+    STICKER_SVG,
```

Comment:
Why does this need a file type? AVATAR has no file type. File kind is not a file type, it specifies semantics, or "what does this file transfer mean, how should it be handled in the client?".